libs/ui-components/src/components/form/validations.ts

```diff
@@ -1,18 +1,33 @@
 import * as Yup from 'yup';
 import { TFunction } from 'i18next';
 
-import { FlightCtlLabel } from '../../types/extraTypes';
+export interface UnvalidatedLabel {
```

Comment:
Lets use `Partial<FlightCtlLabel>` instead of creating a new type

Reply:
Oh, yes. 

I still defined the type in this file to avoid having to define it in each usage, is that OK?